src/cpu-info.spec.ts

```diff
@@ -1,20 +1,18 @@
-import { getCpuInfo } from './cpu-info';
+import { CpuInfo } from './cpu-info';
 
-describe('getCpuInfo', () => {
-  it('should get correct CPU information when available', () => {
+describe('getNumLogicalCores', () => {
```

Comment:
Sorry if I wasn't clear in my previous comment. I meant let's keep `CpuInfo` in the describe block, and in the it blocks, let's do something like `should get the number of logical cores when available`.